package/gluon-web-wifi-config/i18n/fr.po

```diff
@@ -46,3 +46,21 @@ msgstr ""
 "<br /><br />Ici vous pouvez aussi configurer la puissance d'émmission se votre Wi-Fi. "
 "Prenez note que les valeurs fournies pour la puissance de transmission prennent "
 "en compte les gains fournis par l'antenne, et que ces valeurs ne sont pas toujours disponibles ou exactes."
+
+msgid "Outdoor installation"
+msgstr "Installation extérieure"
+
+msgid "Node will be installed outdoors"
+msgstr "FIXME"
```

Comment:
Needs french translation

Reply:
Please just leave missing translations empty, so the original string is used. Missing translations are not blockers for PRs.